docs/docs.less

```diff
@@ -247,7 +249,7 @@ min-version {
     border-color: @gray-300;
     border-style: solid;
     border-width: 0 0 1px;
-    margin: 0;
+    margin: 0 !important;
```

Comment:
Removes extra space showing up below example config. `!important` is used, but we'll refactor once we turn the interactive demo into a single-file Vue component.